.github/workflows/maven-build-oss.yml

```diff
@@ -2,6 +2,26 @@ name: Maven Build OSS
 
 on: 
   workflow_call:
+    inputs:
+      licencecheck:
+        description: 'Need to check licence headers/files'
+        required: false
+        type: boolean
+        default: true
+      sourcepath:
+        description: 'Path to source directory (used for licence check)'
+        required: false
+        type: string
+        default: "src"
+    secrets:
+      NEXUS_MAVEN_REPO:
```

Comment:
urgent: please remove the nexus stuff from the OSS pipeline as no OSS things should have dependencies on closed source code.